recipes/bitserializer/all/conanfile.py

```diff
@@ -120,6 +155,20 @@ def package_info(self):
             self.cpp_info.components["bitserializer-pugixml"].libdirs = []
             self.cpp_info.components["bitserializer-pugixml"].requires = ["bitserializer-core", "pugixml::pugixml"]
 
+        # rapidyaml-archive
+        if self.options.with_rapidyaml:
+            self.cpp_info.components["bitserializer-rapidyaml"].set_property("cmake_target_name", "BitSerializer::rapidyaml-archive")
+            self.cpp_info.components["bitserializer-rapidyaml"].bindirs = []
+            self.cpp_info.components["bitserializer-rapidyaml"].libdirs = []
+            self.cpp_info.components["bitserializer-rapidyaml"].requires = ["bitserializer-core", "ryml::ryml"]
```

Comment:
```suggestion
            self.cpp_info.components["bitserializer-rapidyaml"].requires = ["bitserializer-core", "rapidyaml::rapidyaml"]
```
Component name is not cmake target 😉